app/components/d3plus-scatter.js

```diff
@@ -42,11 +50,18 @@ export default Ember.Component.extend({
     if(this.get('dataType') === 'industries') { return 'rca'; }
   }),
   didInsertElement: function() {
-    Ember.run.scheduleOnce('afterRender', this , function() {
-      this.set('width', this.$().parent().width());
-      this.set('height', this.$().parent().height());
-      this.get('scatter').draw();
-    });
+    $.getJSON(`${apiURL}/data/location?level=department`).then((response) => {
+      let id = this.get('entityId');
+      let data = get(response, 'data');
+      return _.first(_.filter(data, {'year': 2013, 'department_id': parseInt(id) }));
```

Comment:
we need to pull out all these 2013's, etc because in mexico it's different. I'll make an item in jira